docs/snippets/common/storybook-manager-hide-toolbar-on-docs.js.mdx

```diff
@@ -0,0 +1,16 @@
+```js
+// .storybook/manager.js
+
+import { addons } from '@storybook/manager-api';
+
+import startCase from 'lodash/startCase.js';
```

Comment:
Unused import?

Reply:
Thanks, indeed. I copied the file and my IDE didn't catch it with the file being MDX.